kimchi/src/verifier.rs

```diff
@@ -327,8 +304,6 @@ where
         //~~ * poseidon selector
         //~~ * the 15 register/witness
         //~~ * 6 sigmas evaluations (the last one is not evaluated)
-        fr_sponge.absorb_multiple(&public_evals[0]);
-        fr_sponge.absorb_multiple(&public_evals[1]);
```

Comment:
I am still a bit doubtful about not absorbing the public input...

Reply:
We do it when we absorb the other evals

Reply:
Oh I see thanks